molior-deploy.sh.inc

```diff
@@ -268,6 +269,45 @@ create_luks()
   exit_on_error "Error opening LUKS partition $part"
 }
 
+create_raid()
+{
+  if [ -n "$RAID_NAME" ]; then
+    if [ -z "$RAID_DEVICES" ]; then
```

Comment:
This would actually be nice if this could be checked on 'build time' so we get an error when we run `molior-deploy` and not at install time, maybe also add the check to here: https://github.com/molior-dbs/molior-tools/pull/36/files#diff-bef38b2ffbe7604260227c0f5a9ed4910515803bfb1574ea59da3295fb6ea0caR423

Reply:
Good hint @randombenj. Done.